aunt-leahs-app/src/components/locationSelect.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormControl from '@material-ui/core/FormControl';
+
+class LocationSelect extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            locations: [
+                {
+                    name: "Tree Lot",
+                    id: 1,
+                    address: "1234 East Mall"
+                },
+                {
+                    name: "Thrift Store",
+                    id: 2,
+                    address: "1234 Robson St"
+                },
+                {
+                    name: "Fundraising Event",
+                    id: 3,
+                    address: "1234 Wesbrook Mall"
+                }]
+        };
+
+        this.handleLocationChange = this.handleLocationChange.bind(this);
+    }
+
+    handleLocationChange(event) {
+        var selectedId = Number(event.target.value);
```

Comment:
Best practice would be const or let (perhaps const in this case) instead of var

Reply:
Changed, thanks!